packages/rs-sdk/tests/fetch/config.rs

```diff
@@ -22,6 +22,12 @@ const DPNS_DASH_TLD_DOCUMENT_ID: [u8; 32] = [
     68, 209, 224, 212, 242, 106, 141, 142, 255, 55, 207,
 ];
 
+// Variables that change every time we regenerate test vectors
```

Comment:
is it possible to get it from env? we could pass masternode identity then

Reply:
It is read from env. However, for offline testing, we don't have the env, so we must hardcode defaults each time we generate test vectors.